src/vs/workbench/parts/terminal/electron-browser/terminalLinkHandler.ts

```diff
@@ -76,8 +109,8 @@ export class TerminalLinkHandler {
 			this._handleLocalLink(url);
 			return;
 		});
+
 		return this._xterm.registerLinkMatcher(this._localLinkRegex, wrappedHandler, {
-			matchIndex: 1,
```

Comment:
Match index 1 only matches first group of regex, but we need other groups as well which have line and column number information.